src/bp-templates/bp-nouveau/buddypress/search/loop/member-ajax.php

```diff
@@ -24,8 +24,7 @@
 				<p class="item-meta last-activity">
 					<span class="middot">&middot;</span>
 					<?php
-					esc_html_e( 'Last active', 'buddyboss' );
-					echo wp_kses_post( bb_get_member_last_activity_time() );
+					printf( esc_html__( 'Last active %s', 'buddyboss' ), wp_kses_post( bb_get_member_last_activity_time() ) );
```

Comment:
@rajinsharwar2 Adding a space between two strings in this manner is not the correct approach. It may lead to issues when translating the strings.